templates/cfp.html

```diff
@@ -1,390 +1,317 @@
+                      {% include "bootstrap/_field_errors_block.html" with field=form.prior_talks %}
+                      <span class="input-group-addon"><i class="glyphicon glyphicon-globe"></i></span>
+                      {{ form.prior_talks }}
+                   </div>
+                </div>
+             </div>
+             <!-- Text input END-->
+             <!-- Text input START -->
+             <div class="form-group">
+                <label class="col-md-4 control-label">Slides to past talks or current submission</label>
+                <div class="col-md-4 inputGroupContainer">
+                   <div class="input-group">
+                      {% include "bootstrap/_field_errors_block.html" with field=form.file_submission %}
+                      <span class="input-group-addon"><i class="glyphicon glyphicon-upload"></i></span>
+                      {{ form.file_submission }}
+                   </div>
+                </div>
+             </div>
+             <!-- Text input END-->
+             <!-- Submit Button Start -->
+             <div class="form-group">
+                <label class="col-md-4 control-label"></label>
+                <div class="col-md-4">
+                   <button type="submit" class="btn btn-warning">Send <span class="glyphicon glyphicon-send"></span> </button>
+                </div>
+             </div>
+             <!-- Submit Button End -->
+          </fieldset>
+       </form>
+    </div>
+    <!-- Forms End -->
+    <!--Subscribe to stay tune Starts -->
+    <section class="subscribe-2">
+       <div class="container">
+          <div class="row">
+             <div class="col-sm-12 text-center">
+                <i class="icon pe-7s-pen"></i>
+                <i class="icon pe-7s-mail-open-file"></i>
+                <i class="icon pe-7s-paper-plane"></i>
+                <h1 class="large-h1">Subscribe to stay informed</h1>
+             </div>
+          </div>
+          <!--end of row-->
+          <div class="row">
+             <div class="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
+                <form class="email-subscribe mail-list-signup">
+                   <div class="col-sm-7">
+                      <input class="form-email signup-email-field" type="text" placeholder="Enter your email address">
+                   </div>
+                   <div class="col-sm-5">
+                      <input class="btn" type="submit" value="Subscribe">
+                   </div>
+                   <div class="col-sm-12 text-center">
+                      <span>*We never share your email with 3rd parties</span>
+                   </div>
+                   <iframe class="mail-list-form" srcdoc="">
+                   </iframe>
+                </form>
+             </div>
+          </div>
+          <!--end of row-->
+       </div>
+       <!--end of container-->
+    </section>
+    <!--Subsribe to stay Tune Ends -->
 {% endblock content %}
-        <!--Footer Starts-->
```

Comment:
footer here was never rendered, as it's an included template and this content is not in a block @OnurOzayyy